### PR TITLE
Added advance sorting algorithms in sort.py

### LINKED ISSUE #37: Implement more sorting algorithms in existing sort.py
Like:
bitonic_sort.
bogo_sort.py
bucket_sort.py
comb_sort.py.... & many more

### PR DESCRIPTION
**Pull request by Aryamanz29**
-Added bucket, radix, bitonic sort in existing dsalgo/sort.py
-Added test in sort_tests.py
closes #37

### CODE REVIEW

File: dsalgo/sort.py [hunk 1]
```diff
@@ -31,8 +31,12 @@ def __new__(self, array, algo, reverse=False):
             return bubble_recursion(self.array,self.reverse)
         if self.algo =='selection':
             return selection(self.array,self.reverse)
-        if self.algo =='quick':
```
Comment (codesankalp) [inline dsalgo/sort.py:34]: why do you remove quick sort?
revert this back.

File: dsalgo/sort.py [hunk 2]
```diff
@@ -201,3 +205,120 @@ def partition(array, start, end):
...
+def next_sort(arr):
+     # We will use insertion sort here.
+     for i in range(1, len(arr)):
+         j = i - 1
+         key = arr[i]
+         while arr[j] > key and j >= 0:
+             arr[j+1] = arr[j]
+             j = j - 1
+         arr[j + 1] = key
+     return arr
```
Comment (codesankalp) [inline dsalgo/sort.py:279]: @Aryamanz29 aryaman use flake 8 to test the pep8 guidelines.
you haven't followed pep8 guidelines properly

**Review by codesankalp — CHANGES_REQUESTED** [review]
Do the required changes @Aryamanz29

File: dsalgo/sort.py (hunk 2 above)
Comment (Aryamanz29) [reply dsalgo/sort.py:279]: okay

**Review by codesankalp — APPROVED** [review]

**Comment by codesankalp** [comment]
@devkapilbansal please review it.

File: dsalgo/sort.py [hunk 3]
```diff
@@ -1,9 +1,9 @@
 import sys
 
-class Sort:
 
+class Sort:
     def __new__(self, array, algo, reverse=False):
-        '''
+        """
```
Comment (devkapilbansal) [inline dsalgo/sort.py:6]: Use single quotes as said earlier

**Comment by devkapilbansal** [comment]
Will review it again after merging #49

**Pull request closed without merging**